installer/src/config/resources/unix_shortcut_specification.xml

```diff
@@ -2,19 +2,19 @@
 	<defaultCurrentUser/>
 	<programGroup defaultName=""
 		location="applications"
-		iconFile="$INSTALL_PATH/studio/48-jbds_icon.png"
+		iconFile="$INSTALL_PATH/studio/48-devstudio.icon.png"

```

Comment:
should be 48-devstudio_icon.png here and in several places below for consistency